French Translation Updated

SteamWind opened the pull request:
Completing and updating the French translation.

Commit:
French Translation Updated

Completing and updating the French translation

fletch3555 commented on publishable/lang/fr/database.php:
```diff
@@ -51,30 +44,31 @@
-        'which_column_from'    => 'Quelle colonne du',
-        'is_used_to_reference' => 'est utilisée pour référencer le',
-        'pivot_table'          => 'Tableau croisée dynamique (Pivot Table)',
+        'has_one'              => null,
```
Any particular reason these are null?

SteamWind replied:
Yes, because it shouldn't be translated... 
Null = use default language
Not present = not translated

fletch3555 replied:
@emptynick tested it and it looks like null value and missing key behave the same.  Can you verify this and remove the entry? I feel the null value may get confusing and difficult to maintain long-term.  I'd rather have it be missing entirely, which indicates it needs a translation (or should be left alone)

SteamWind replied:
For me, not having a value or value set on Null use the default translation (EN). That's weird.
Anyway I commit the translation of those values in EN.

Commit:
Forgotten plural of user

Forgotten plural of user

Commit:
Translating null value

Changing new value as requested in review: https://github.com/the-control-group/voyager/pull/3269#discussion_r192536520

SteamWind commented:
Hey, does I missed something? Is it okay to merge those translations?
I made the changes on the null values.

fletch3555 committed:
Merge branch '1.1' into feature-french-translation-update
